src/main/java/homework_4/custom_file_reader/CustomFileReader.java

```diff
@@ -0,0 +1,125 @@
+package homework_4.custom_file_reader;
+
+import homework_4.custom_annotation.CustomAnnotation;
+
+import java.io.*;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+public class CustomFileReader {
```

Comment:
Doesn't fit the requirement from Homework chat 
![image](https://user-images.githubusercontent.com/58872509/134572175-e926cb88-39f4-4fa5-87e7-3c6c72c06bde.png)

Reply:
Missed this requirement..  updated